PowershellTools.ProcessManager.Client/ProcessManagement/PowershellHostProcessFactory.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using PowershellTools.ProcessManager.Data.Common;
+
+namespace PowershellTools.ProcessManager.Client.ProcessManagement
+{
+    internal static class PowershellHostProcessFactory

```

Comment:
Will do this later.

From: Andre Sayre [mailto:notifications@github.com]
Sent: Wednesday, January 14, 2015 6:32 PM
To: Microsoft/poshtools
Cc: Huanhuan Sun
Subject: Re: [poshtools] Infrastructure for out-of-proc, includes initializing a process and creating a WCF channel for servicing (#36)


In PowershellTools.ProcessManager.Client/ProcessManagement/PowershellHostProcessFactory.cs<https://github.com/Microsoft/poshtools/pull/36#discussion-diff-22987269>:

> @@ -0,0 +1,122 @@

> +﻿using System;

> +using System.Collections.Generic;

> +using System.Diagnostics;

> +using System.Globalization;

> +using System.IO;

> +using System.Linq;

> +using System.Reflection;

> +using System.Text;

> +using System.Threading;

> +using System.Threading.Tasks;

> +using PowershellTools.ProcessManager.Data.Common;

> +

> +namespace PowershellTools.ProcessManager.Client.ProcessManagement

> +{

> +    internal static class PowershellHostProcessFactory

Comments on the classes and public methods where they make sense would be good!

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/poshtools/pull/36/files#r22987269>.